package.json

```diff
@@ -24,44 +24,54 @@
   },
   "dependencies": {
     "classnames": "*",
+    "create-react-class": "^15.6.3",
     "deepmerge": "^1.4.0",
     "lodash.debounce": "4.0.3",
     "lodash.values": "4.1.0",
-    "react": "0.14.7",
-    "react-dom": "0.14.7",
```

Comment:
Looks like we were only 16 major versions behind :)

Reply:
We should've waited until React 16.4.7 was released prior to the update to reach that exact 16 major versions behind goal